### PR TITLE
ProjectList: update to match current semester teams' repository

### PR DESCRIPTION
**Pull request by eugenepeh**

### CODE REVIEW

**Comment by eugenepeh** [comment]
@sooyj @okkhoy @hwkchia, for your review.

File: admin/projectList.md [hunk 1]
```diff
@@ -213,27 +213,27 @@
 <panel header="T10" expanded>
 
 
-### [B1](http://github.com/CS2103AUG2017-T10-B1/main)
+### [B1](http://github.com/CS2103JAN2018-T10-B1/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no T10 this time. Please remove them

File: admin/projectList.md [hunk 2]
```diff
@@ -274,27 +274,27 @@
 <panel header="T12" expanded>
 
 
-### [B1](http://github.com/CS2103AUG2017-T12-B1/main)
+### [B1](http://github.com/CS2103JAN2018-T12-B1/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no T12

File: admin/projectList.md [hunk 3]
```diff
@@ -304,15 +304,15 @@
...
-### [B1](http://github.com/CS2103AUG2017-T13-B1/main)
+### [B1](http://github.com/CS2103JAN2018-T13-B1/main)
 
-<img src="https://rawgit.com/CS2103AUG2017-T13-B1/main/master/docs/images/Ui.png" width="600"/>
+<img src="https://rawgit.com/CS2103JAN2018-T13-B1/main/master/docs/images/Ui.png" width="600"/>
 
 <hr>
 
-### [B2](http://github.com/CS2103AUG2017-T13-B2/main)
+### [B2](http://github.com/CS2103JAN2018-T13-B2/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no T13

File: admin/projectList.md [hunk 4]
```diff
@@ -323,27 +323,27 @@
...
-### [B3](http://github.com/CS2103AUG2017-T14-B3/main)
+### [B3](http://github.com/CS2103JAN2018-T14-B3/main)
 
-<img src="https://rawgit.com/CS2103AUG2017-T14-B3/main/master/docs/images/Ui.png" width="600"/>
+<img src="https://rawgit.com/CS2103JAN2018-T14-B3/main/master/docs/images/Ui.png" width="600"/>
 
 <hr>
 
-### [B4](http://github.com/CS2103AUG2017-T14-B4/main)
+### [B4](http://github.com/CS2103JAN2018-T14-B4/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no T14

File: admin/projectList.md [hunk 5]
```diff
@@ -415,27 +415,27 @@
 <panel header="T17" expanded>
 
 
-### [B1](http://github.com/CS2103AUG2017-T17-B1/main)
+### [B1](http://github.com/CS2103JAN2018-T17-B1/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no T17

File: admin/projectList.md [hunk 6]
```diff
@@ -473,27 +473,27 @@
 <panel header="F10" expanded>
 
 
-### [B1](http://github.com/CS2103AUG2017-F10-B1/main)
+### [B1](http://github.com/CS2103JAN2018-F10-B1/main)
```
Comment (okkhoy) [inline admin/projectList.md]: There is no F10

File: admin/projectList.md [hunk 7]
```diff
@@ -504,27 +504,27 @@
...
-<img src="https://rawgit.com/CS2103AUG2017-F11-B3/main/master/docs/images/Ui.png" width="600"/>
+<img src="https://rawgit.com/CS2103JAN2018-F11-B3/main/master/docs/images/Ui.png" width="600"/>
 
 <hr>
 
-### [B4](http://github.com/CS2103AUG2017-F11-B4/main)
+### [B4](http://github.com/CS2103JAN2018-F11-B4/main)
 
-<img src="https://rawgit.com/CS2103AUG2017-F11-B4/main/master/docs/images/Ui.png" width="600"/>
+<img src="https://rawgit.com/CS2103JAN2018-F11-B4/main/master/docs/images/Ui.png" width="600"/>
```
Comment (okkhoy) [inline admin/projectList.md]: Please add F12 and F14

**Review by okkhoy — CHANGES_REQUESTED** [review]
Hi Eugene,
Thanks for this! Since you have the groundwork done, can you update the PR with the necessary changes indicated?

Thanks,
Akshay

**Comment by eugenepeh** [comment]
@okkhoy made changes as requested. Thanks for the heads up, forgot that tutorial slots changes.

**Review by okkhoy — APPROVED** [review]

**Pull request merged**